src/api/parser/src/utils/supabase.js

```diff
@@ -25,7 +25,10 @@ module.exports = {
 
     return data.map((feed) => ({
       // Prefer the a user's display name if present, fallback to wiki name otherwise
-      author: feed.display_name || feed.wiki_author_name,
+      author:
```

Comment:
Let's use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to simplify this a bit:

```js
author: feed.telescope_profiles?.display_name || feed.wiki_author_name,
```